commons/connector/health/health_appleExerciseTime.avsc

```diff
@@ -0,0 +1,24 @@
+{
+  "namespace": "org.radarcns.connector.health",
+  "name": "HKQuantityTypeIdentifierAppleExerciseTime",
+  "type": "record",
+  "doc": "AppleExerciseTime , unit is min.",
```

Comment:
Does `apple` here stand for the Apple watch?

Reply:
In the record doc, there's no need to add the unit, please add it to the field doc instead. In the record doc, you can document how these records are obtained (i.e. via Apple HealthKit).